### ISSUE TITLE
Double open dialog appearing when importing SVG

### ISSUE THREAD

**Issue by romannurik** [alexjlockwood/ShapeShifter#237]
When performing an SVG import, I'm finding that the system file picker appears twice, seemingly discarding the file selected the first time :-/

**Comment by alexjlockwood** [comment]
Yeah, this seriously annoys me too... I haven't looked into it too much but at first glance I had no idea what was going on and/or why it was happening. IIRC getting the `<input>`s to work was a huge pain to setup and I wouldn't be surprised if I did something hacky/not correct. If you have any ideas... lmk. :)

**Comment by alexjlockwood** [comment]
Actually, I think the issue is because for some reason my button's click listener is invoked twice... hmm

**Comment by alexjlockwood** [comment]
a-ha, I think I figured it out. I was missing an `event.preventDefault()` I think. :)

**Comment by romannurik** [comment]
👍👍👍 nice!
